apis/api_utils/api_utils/api/shared.py

```diff
@@ -468,105 +467,110 @@ def create_uhs_download_zip(
 
 
 def write_gms_download_data(
-    gms_result: sc.gms.GMSResult,
+    gms_result: gc.gms.GMSResult,
     out_dir: str,
-    disagg_data: sc.disagg.EnsembleDisaggResult,
-    cs_param_bounds: sc.gms.CausalParamBounds = None,
+    disagg_data: gc.disagg.EnsembleDisaggResult,
     prefix: str = None,
 ):
     prefix = "" if prefix is None else f"{prefix}_"
 
+    # Get the waveforms
     missing_waveforms = gms_result.gm_dataset.get_waveforms(
         gms_result.selected_gms_ids, gms_result.site_info, out_dir
     )
+    if len(missing_waveforms) > 0:
+        print(
+            f"Failed to find waveforms for the following records:\n{missing_waveforms}"
+        )
```

Comment:
Was a bit confused at first about what this was doing as the get was returning missing data.
Now that I've read the function get_waveforms this makes sense.a
Not sure if the comment should be changed slightly or the docstring of get_waveforms as the function name get_waveforms make some think I will get returned the waveforms not missing waveforms